### ISSUE TITLE
Flying spell debuff affecting player in bladestorm

### ISSUE THREAD

**Issue by lucasfernkr** [TrinityCore/TrinityCore#26457]
**Description:**

When a player casts an flying spell ( like frostbolt, or warlock death coil ) and a warrior casts bladestorm when the spells is flying to him, the debuff affects the warrior and slows him/cc him while he is in bladestorm.

**Expected behaviour:**

The warrior would not have to be affected by those CCs even if the spell is already casted and flying into the warrior.

**Steps to reproduce the problem:**

1. Cast Death Coil / Frostbolt / Any spell that is a projectile after its casted and had a CC
2. Cast Bladestorm(46924) with the warrior prior to the spell hits you

**Branch(es):**

3.3.5 branch

**TC rev. hash/commit:** 

1c0029321a84

**Operating system:** 

Windows 10

**Comment by offl** [comment]
I'm pretty sure I saw an issue about this generic bug

**Comment by lucasfernkr** [comment]
Yeah, seems pretty generic beacuse you can reproduce it on similar scenarios.
